index.html

```diff
@@ -16,9 +16,9 @@
     <meta property="og:url" content="#">
     <meta property="og:description" content="Setting up a default portfolio according to template.">
 
-    <link rel="stylesheet" href="css/main.css">
+    <!-- <link rel="stylesheet" href="css/main.css">
     <link rel="stylesheet" href="css/dropmenu.css">
-    <link rel="stylesheet" href="css/media-query.css">
+    <link rel="stylesheet" href="css/media-query.css"> -->
```

Comment:
- [ ] I think you can uncomment now the stylesheets. This is a good resource to verify that the readability and structure of the page make sense, but we don't want our page to go into production without styles.

Reply:
- [ ] Let's make sure that each one of the inputs in our form has an associated label to it like this:
> Every form control has a label associated with it using 'label', 'for', and 'id'.

This are going to be useful for users using screen readers. Since we only want them to see them, there are some tricks to not display them in the screen, but they will still be read by the screen reader device. [Read more about it here.](https://css-tricks.com/inclusively-hidden/)